index.bs

```diff
@@ -0,0 +1,221 @@
+<pre class=metadata>
+Title: Layout Instability API
+Status: ED
+Shortname: layout-instability
+Group: WICG
+Level: 1
+Editor: Nicolás Peña Moreno, Google, https://google.com, npm@chromium.org
+        Steve Kobes, Google, https://google.com, skobes@chromium.org
+        Emily Hanley, Google, https://google.com, eyaich@chromium.org
+URL: https://wicg.github.io/layout-instability
```

Comment:
I think this link won't work until we move to WICG.

Reply:
Removed for now. I noted to add it back in later.